docs/js/redux-injectors.md

```diff
@@ -0,0 +1,27 @@
+# `redux-injectors`
```

Comment:
Are you linking to this file from anywhere else in the docs?

Reply:
You might want to or very few people might ever read it.

Reply:
Linked to it from `js/README.md`